alacritty/src/display.rs

```diff
@@ -234,7 +235,7 @@ impl Display {
             glyph_cache,
             meter: Meter::new(),
             size_info,
-            font_size: config.font.size,
+            font: config.font.clone(),
```

Comment:
This clone is problematic, since it means the `font` exist twice. Both in the event processor and the display. This makes it very difficult to keep track of state.

We should check if it is even needed at all or if we can just keep all the state on the event processor.